Pairwise_potential.py

```diff
@@ -74,12 +75,20 @@ def cutoff_correction(self, cutoff, box_object, num_particles,):
     """
 
         volume = box_object.volume
-        sig_by_cutoff3 = np.power(self.sigma/cutoff, 3)
+        sig_by_cutoff3 = np.power(self.sigma/cutoff2, 3)
```

Comment:
We still need to use `cutoff` here but we only have `self.cutoff2` so we first need to calculate `np.sqrt(self.cutoff2)`.